LocationServices/LocationServicesTests/MGLCoordinateBoundsExtensionTests.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  MGLCoordinateBoundsExtensionTests.swift
+//  MLNCoordinateBoundsExtensionTests.swift
```

Comment:
Should we rename the file to match the code inside?

Reply:
renamed